dvl-a50/static/js/rotatedmarker.js

```diff
@@ -0,0 +1,57 @@
+(function() {
```

Comment:
It would be nice to move to a typescript project

Reply:
yeah, this is an external lib though.